src/module/actor/creature/data.ts

```diff
@@ -91,7 +91,7 @@ interface CreatureSystemData extends Omit<CreatureSystemSource, "attributes">, A
     /** Saving throw data */
     saves: CreatureSaves;
 
-    skills: Record<string, SkillData>;
+    skills: Record<SkillLongForm, SkillData>;
```

Comment:
Where did lore go?

Reply:
Character already doesnt account for lore in the type. That said, i could see about improving the type.

Reply:
I think the only other option is `string`, unfortunately.

Reply:
Record<SkillLongForm, SkillData> & Record<string, SkillData | undefined> wouldn't work?